src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -171,3 +172,23 @@ Napi::Value ModelWrap::get_output_shape(const Napi::CallbackInfo& info) {
         return info.Env().Undefined();
     }
 }
+
+Napi::Value ModelWrap::clone(const Napi::CallbackInfo& info) {
+    Napi::Env env = info.Env();
+    if (info.Length() > 0) {
+        reportError(env, "clone() does not accept any arguments.");
+        return env.Undefined();
+    }
+    return ModelWrap::wrap(env, _model->clone());
+}
+
+Napi::Object ModelWrap::wrap(Napi::Env env, const std::shared_ptr<ov::Model> model) {
+    Napi::HandleScope scope(env);
+    const auto& prototype = env.GetInstanceData<AddonData>()->model;
+    if (!prototype) {
+        OPENVINO_THROW("Invalid pointer to Model prototype.");
+    }
+    auto obj = prototype.New({});
+    Napi::ObjectWrap<ModelWrap>::Unwrap(obj)->set_model(model);
+    return obj;
+}
```

Comment:
This is almost the same as `Napi::Object cpp_to_js(const Napi::Env& env, std::shared_ptr<ov::Model> model)` from `helper.hpp`
Please remove it

Reply:
Done and using the same when returning the cloned model